examples/simpack_xrd_dyn_AlGaAs.py

```diff
@@ -0,0 +1,178 @@
+# This file is part of xrayutilities.
+#
+# xrayutilities is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Copyright (c) 2016-2023 Dominik Kriegner <dominik.kriegner@gmail.com>
+
+
+
+# This is an example of the modelling and fitting of the diffraction data obtained on epitaxial heterostructure
+# The design is as follow
+#               _________________________
+#
+#                       GaAs 320 nm          using the same Ga cell as GaAs below (same growth rate)
+#               _________________________
+#
+#                  Al_0.80GaAs 1.8 micron    using the same Ga & Al cells as Al_0.80GaAs below (same growth rate and composition)
+#               _________________________
+#
+#                       GaAs 200 nm           
+#               _________________________
+#
+#                  Al_0.80GaAs 2 micron
+#               _________________________
+#
+#              GaAs substrate and buffer (001)
+#
+# Xray set-up: X'PERT Panalytical with
+#       copper anode,focusing mirror, Ge(220)x4 monochromator, Ge(220)x2 triple axis detection
+# Copyright (c) 2023 C2N - CNRS Author:	  Aristide Lemaître <aristide.lemaitre@c2n.upsaclay.fr>
+
+
+
+import xrayutilities as xu
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def x_Al(aper,apar,mat):
+    # determination of Al composition from c parameter, poisson coefficient and Veegard's law
```

Comment:
Vegard's "law" is not a law. In fact its the most simply assumption and often shown to be not precisely correct. I therefore prefer to call it Vegard's rule.
Also I think the `AlGaAs.ContentBsym` function should already implement this functionality.